pkg/semtech-loramac/contrib/semtech_loramac.c

```diff
@@ -0,0 +1,624 @@
+/*
+ * Copyright (C) 2017 Inria
+ *               2017 Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ * @ingroup     pkg_semtech-loramac
+ *
+ * @file
+ * @brief       Implementation of public API for Semtech LoRaMAC
+ *
+ * This implementation is an adaption of the applications provided on the
+ * Semtech Lora-net repository.
+ *
+ * The LoRaMAC stack and the SX127x driver run in their own thread and simple
+ * IPC messages are exchanged to control the MAC.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @author      Jose Alamos <jose.alamos@inria.cl>
+ * @}
+ */
+
+#include <string.h>
+
+#include "msg.h"
+#include "shell.h"
+
+#include "semtech_loramac.h"
+
+#include "sx127x.h"
+#include "sx127x_params.h"
+#include "sx127x_netdev.h"
+
+#include "net/netdev.h"
+#include "net/loramac.h"
+
+#include "semtech-loramac/board.h"
+#include "LoRaMac.h"
+#include "region/Region.h"
+#include "utilities.h"
```

Comment:
is that header includable with a path? ```utilities.h``` by itself sounds generic enough that it might lead to clashes otherwise.